lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -122,51 +157,57 @@ void testForceCompleteNotClaimed() throws Exception {
 
```

Comment:
General note regarding the exceptions:
Since TSK-1647 our exceptions contain some information.
Should we modify the assert statements accordingly in this ticket?